package.json

```diff
@@ -79,8 +79,8 @@
     "tui-release-notes": "github:nhn/tui.release-notes#v1.0.0",
     "typescript": "^3.2.2",
     "webpack": "^3.10.0",
-    "webpack-bundle-analyzer": "^2.9.1",
-    "webpack-dev-server": "^2.10.0"
+    "webpack-bundle-analyzer": ">=3.3.2",
+    "webpack-dev-server": ">=3.1.11"
```

Comment:
Why did you use `>=`? Can you change to using the carat `^` for no specific reason?

Reply:
I put in what composer said to put in LOL.
I'll make the change.